### PR TITLE
add categories type for TASKs

### PR DESCRIPTION
**Pull request by dolibit-ut**
first step to add categories for tasks

### CODE REVIEW

File: htdocs/categories/class/categorie.class.php [hunk 1]
```diff
@@ -151,7 +155,8 @@ class Categorie extends CommonObject
 		'actioncomm' => 'ActionComm',
 		'website_page' => 'WebsitePage',
 		'ticket' => 'Ticket',
-		'knowledgemanagement' => 'KnowledgeRecord'
+		'knowledgemanagement' => 'KnowledgeRecord',
+		'task' => 'Task';
```
Comment (frederic34) [inline htdocs/categories/class/categorie.class.php:159]: @dolibit-ut syntax error here

Comment (frederic34) [reply htdocs/categories/class/categorie.class.php:159]: @dolibit-ut can you fix the syntax error please ?

**Comment by eldy** [comment]
@dolibit-ut can you fix conflict ?

**Comment by eldy** [comment]
It seems this PR was set with "Discussion" or "PR to fix" tag, but no answer nor correction was provided since a long time. So request is automatically closed.
Please reopened and add a comment if you think this is an error.

**Pull request closed without merging**